src/main/java/ui/components/NavigableListView.java

```diff
@@ -45,7 +46,9 @@
     // (when nothing is selected, both should be no-ops)
     private boolean saveSelectionCalled = false;
 
-    private IntConsumer onItemSelected = i -> {};
+    public static final boolean SHOULD_LOAD_BVIEW_ISSUE = true;
```

Comment:
Details about what this field is going to be used for shouldn't leak into this class. This just indicates whether or not the selection was caused by a right-click, right?

Actually, is there a case where we want a right click to trigger a selection event? If not, maybe we don't even need to change this.